### PR TITLE
Create Main.java changed folder test

### PR DESCRIPTION
**Pull request by think-shark**

### CODE REVIEW

**Comment by RajkumarKarnakar** [comment]
ok add

**Review by think-shark — COMMENTED** [review]
approved

**Comment by RajkumarKarnakar** [comment]
merge main

**Comment by RajkumarKarnakar** [comment]
added new Main file codes

**Review by think-shark — COMMENTED** [review]
done

**Comment by RajkumarKarnakar** [comment]
done

**Comment by RajkumarKarnakar** [comment]
done

File: simple_codes/Main.java
Comment (RajkumarKarnakar) [inline simple_codes/Main.java:1]: corrected

**Review by think-shark — COMMENTED** [review]
checked and tested

File: simple_codes/Main.java
Comment (RajkumarKarnakar) [reply simple_codes/Main.java:1]: ok

**Pull request merged**